src/WpfViews/src/TasExplorerView.xaml

```diff
@@ -70,14 +70,14 @@
         <StackPanel Orientation="Horizontal" Grid.Row="0" Grid.ColumnSpan="3" Background="{Binding ElementName=TasExplorerViewElement, Path=WizardFooterBrush}">
             <Button Style="{DynamicResource RoundedButtonStyle}"
                     Command ="{ Binding ElementName=TasExplorerViewElement, Path=OpenLoginFormCommand }" CommandParameter="{Binding RelativeSource={RelativeSource AncestorType=Window}}">
-                <Image Source="Resources/AddCloud.png" Stretch="Uniform" Width="16px" Height="16px"
+                <Image Source="Resources/tanzu_nc_dark.png" Stretch="Uniform" Width="16px" Height="16px"
                        ToolTip="Target New Tanzu Application Service"/>
             </Button>
 
             <Button Style="{DynamicResource RoundedButtonStyle}"
                     Command ="{ Binding ElementName=TasExplorerViewElement, Path=RefreshAllCommand }">
                 <Grid>
-                    <Image Source="Resources/Refresh.png" 
+                    <Image Source="Resources/Refresh_16.png" 
```

Comment:
I misspelled the name of this image -- this is a self reminder to fix this typo before merging

```suggestion
                    <Image Source="Resources/Refresh_16x.png" 
```

Reply:
Resolved by 4c4cf68